### PR TITLE
Performing GLM Analysis

### PR DESCRIPTION
**Pull request by YolaKamalita**
Solving #3 

Collaborators:
@Xiaoyu0077 
@YolaKamalita 

What's Done?
- Model fitting
- Model Selection
- Finalize description
- Add conclusion

### CODE REVIEW

File: quarto/Group_10_Analysis.qmd [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+## Model Selection
+
+```{r export_summary, results='asis'}
+
+#| echo: true
+#| warning: false
+#| error: false
+
+# Comparing models with different explanatory variables
```
Comment (chelsea7711) [inline quarto/Group_10_Analysis.qmd]: Showing two models the full model, removing year+cotes

Comment (Xiaoyu0077) [reply quarto/Group_10_Analysis.qmd]: I retain model2(full model) and 3(remove year and votes).

File: quarto/Group_10_Analysis.qmd [hunk 2]
```diff
@@ -36,153 +38,295 @@ Description:
...
+               family = binomial(link = "logit"))
+model_3 <- glm(rating_higher_than_7 ~ length + budget + genre, 
+               data = df, 
+               family = binomial(link = "logit"))
+
+suppressWarnings(export_summs(model, model_2, model_3,
+                error_format = "[{conf.low}, {conf.high}]"))
+```
+
+After doing some investigations, the model without year (model 1) is the one that can give the lowest AIC. Removing votes (model 3) does not make the AIC becoming lower and BIC is also only resulted in a slightly lower. Hence, model 1 is selected to be the final model.
```
Comment (Xiaoyu0077) [inline quarto/Group_10_Analysis.qmd]: Model 3 is selected to be the final model.

File: quarto/Group_10_Analysis.qmd [hunk 3]
```diff
@@ -36,153 +38,295 @@ Description:
...
+
```
Comment (Xiaoyu0077) [inline quarto/Group_10_Analysis.qmd:565]: Add two new models for rating higher than 7 and lower than 7 separately.

**Review by chelsea7711 — APPROVED** [review]

**Review by Xiaoyu0077 — APPROVED** [review]

**Review by M-MWW — APPROVED** [review]

**Pull request merged**